fvm/evm/stdlib/contract.go

```diff
@@ -1022,7 +1075,7 @@ var internalEVMTypeCallFunctionType = &sema.FunctionType{
 			TypeAnnotation: sema.NewTypeAnnotation(sema.UIntType),
 		},
 	},
-	ReturnTypeAnnotation: sema.NewTypeAnnotation(sema.ByteArrayType),
+	ReturnTypeAnnotation: sema.NewTypeAnnotation(sema.AnyStructType),
```

Comment:
Change to `Result`

Reply:
Hmm, how do we set a type annotation for a struct like `Result` ?